packages/yoastseo/spec/parse/traverse/findAllInTreeSpec.js

```diff
@@ -0,0 +1,43 @@
+import findAllInTree from "../../../src/parse/traverse/findAllInTree";
+import build from "../../../src/parse/build/build";
+import LanguageProcessor from "../../../src/parse/language/LanguageProcessor";
+import Factory from "../../specHelpers/factory";
+
+describe( "A test for findAllInTree", () => {
+	it( "should correctly extract all p-tags from a tree", function() {
+		const html = "<div><p class='yoast'>Hello, world! </p><p class='yoast'>Hello, yoast! </p></div>";
+
+		const researcher = Factory.buildMockResearcher( {} );
+		const languageProcessor = new LanguageProcessor( researcher );
```

Comment:
This is a little bit repetitive 😄 I'm going to move these two lines into a `beforeEach`. See https://jestjs.io/docs/setup-teardown for documentation.